docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+Getting started from Scratch
+----------------------------
+
+**Table of Contents**
+
+@@TOC@@
+
+### Intended Audience
+
+This guide is for people who want to craft a custom Kubernetes cluster.  If you
+can find an existing Getting Started Guide that meets your needs on [this
+list](README.md), then we recommend using it, as you will be able to benefit
+from the experience of others.  However, if you have specific IaaS, networking,
+configuration management, or operating system requirements not met by any of
+those guides, then this guide will provide an outline of the steps you need to
+take.  Note that it requires considerably more effort than using one of the
+pre-defined guides.
+
+This guide is also useful for those wanting to understand at a high level some of the
+steps that existing cluster setup scripts are making.
+
+### Prerequisites
```

Comment:
I don't want to call those prerequisites, because that implies that people should go prepare those things before working further.  But, I don't want to them to do that.  I want them to read further to understand the different ways in which they could do those things before doing them.